src/full/Agda/TypeChecking/Primitive/Cubical.hs

```diff
@@ -272,7 +272,17 @@ doPiKanOp cmd t ab = do
             Def q [] | Just q == mInterval -> return $ Just $ \_ _ a0 -> a0
             _ -> return Nothing
 
-        _ -> return Nothing
+        Prop _     -> pure Nothing
+        SSet _     -> pure Nothing
+        Inf _ _    -> pure Nothing
+        LevelUniv  -> pure Nothing
```

Comment:
correct?

Reply:
Yes, this matches `isCoFibrantSort`, we only return `Just` here for the `True` cases there: that's when a pi type should support kan operations.

Reply:
Perhaps it would make sense to tie this code more closely to `isCofibrantSort` (or at least to add some documentation of the connection).